CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
```

Comment:
In the CodeChecker repository we do not track changes in a file but we are writing release notes (https://github.com/Ericsson/codechecker/releases) and collecting PR/issues in milestones (https://github.com/Ericsson/codechecker/milestones). If you think this file has some benefit in this repo we can keep it (in this case it should be updated every time), otherwise it should be removed.

Reply:
On the VS Marketplace page, a changelog is directly linked from the version history page, if it exists.
For example here's the [GitLens extension's version history](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens&ssr=false#version-history).
As far as I know, this doesn't recognize Github's Releases page.

Even if we don't keep a detailed changelog of the project, an auto-export of the GitHub Releases should be considered for it.

Reply:
I didn't know about it. Lets keep this file in the repo then.